pkg/cache/session_store.go

```diff
@@ -0,0 +1,46 @@
+package cache
+
+import (
+	"time"
+
+	"github.com/CESARBR/knot-babeltower/pkg/network"
+)
+
+// Cacher represents the operations to send commands response
+type SessionStore interface {
+	Get(email string) (string, error)
+	Save(email string, id string) error
+}
+
+const SESSION_EXPIRATION = "24h"
```

Comment:
Would it make sense for this to be set by an environment variable? :thinking:

Reply:
Probably.

Reply:
Done.